src/translations/lang/en.json

```diff
@@ -208,22 +210,22 @@
 		"DATA": "Data",
 		"DECRYPT_ACCESS": "How would you like to access your assets? ",
 		"DECRYPT_PROMPT_SELECT_ADDRESS": "Select an Address",
-		"DEFIZAP": "DeFiZap",
-		"DEFIZAP_QUESTION_FIVE": "How is the MyCrypto DeFiZap integration different from the functionality of DeFiZap.com?",
-		"DEFIZAP_QUESTION_FIVE_ANSWER": "The MyCrypto DeFiZap integration uses the same process as the DeFiZap.com process. MyCrypto’s DeFiZap integration adds an extra layer of convenience by allowing you to access Zaps without having to leave MyCrypto. For the full list of available Zaps, visit [DeFiZap.com](https://defizap.com).",
+		"DEFIZAP": "Zapper",
```

Comment:
Why is this part of this PR?

Reply:
because rebasing a sorted json file is a fucking nightmare xD